### DISCUSSION TITLE
ICS subscription vs. one-time event insertion

### DISCUSSION THREAD

**Discussion by thomaskonrad** [solspace/craft-calendar#55] [Questions & Answers] [answered]
When clicking an ICS link for a calendar in the plugin, for some reason, iOS wants to add the contained events only once. I had other ICS links that were a subscription instead of just a one-time add. I don't know exactly what's the difference between the ICS files that causes that difference, but there must be one. It would be awesome to have the choice whether it's a subscription which is regularly polled, or a one-time addition to a local calendar.

**Comment by kjmartens** [comment, marked as answer]
I think you might be misunderstanding how this is supposed to work. There are currently 2 different approaches:

1. ICS Export - this allows users to download and open an ICS file that then opens up into their calendaring app, importing the events once.
2. ICS Subscriptions - in order for this to work, you need to provide the user with the URL Calendar provides (when enabled), and they need to specifically open up their calendaring app, create a new calendar subscription, and paste that URL in there. If you attempt to have your users just visit the URL in their browser, it will behave like (1) above (ICS Export) and import the events once. :)

More information about this can be found here:
https://docs.solspace.com/craft/calendar/v3/overview/exporting.html
